src/main/scala/io/iohk/ethereum/domain/Blockchain.scala

```diff
@@ -80,6 +80,13 @@ trait Blockchain {
   def getMptNodeByHash(hash: ByteString): Option[MptNode]
 
 
+  def getAccount(address: Address): Option[Account] = {
+    val hash = crypto.kec256(address.bytes)
+    getMptNodeByHash(hash).collect {
```

Comment:
I'm wondering, for getting the account given a certain address shouldn't we use a MPT and use state root hash of the best block?